src/game/objects/har.c

```diff
@@ -765,8 +785,6 @@ void har_take_damage(object *obj, const str *string, float damage, float stun) {
             obj->vel.y = -7 * object_get_direction(obj);
             h->state = STATE_FALLEN;
             object_set_stride(obj, 1);
-        } else {
-            object_set_custom_string(obj, str_c(string));
```

Comment:
This seems wrong, why would you not use the 'hit' animation string in this case?

Reply:
oh you do it above, nm